dev-docs/bidders/mediaConsortium.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: bidder
+title: Media Consortium
+description: Prebid Media Consortium Bidder Adapter
+biddercode: mediaConsortium
+media_types: banner, video
+pbjs: true
+userId: all
+multiformat_supported: will-bid-on-any
+sidebarType: 1
+---
+
+### Prebid JS
+
+To get acces to the full feature set of the adapter you'll need to allow localstorage access in the `bidderSettings`.
+
+```javascript
+    pbjs.bidderSettings = {
+        mediaConsortium: {
+            storageAllowed: true
+        }
+    }
+```
+
+#### Controlling 1plusX profile API usage and FPID retrieval
```

Comment:
What is 1plusX?

Reply:
It is a [DMP](https://www.1plusx.com/), we retrieve the 1plusX userId from the localstorage in the adapter (if it exists) and we retrieve various user/site specific segments in the backend using data from the bidrequest

Reply:
Thanks for the other changes. This 1plusX thing is the last item - I think this section needs some kind of transition. For example "MediaConsortium recommends the use of the [1plusX real time data module](/dev-docs/modules/1plusXRtdProvider.html). After you include the module ... " then the rest of the section makes more sense.

Reply:
I've updated the section title and added a transition, let me know if it's OK for you